### DISCUSSION TITLE
WASM Support?

### DISCUSSION THREAD

**Discussion by MarkOverstreet** [icerpc/icerpc-csharp#3959] [Q&A] [answered]
### Desired feature

I don't see a place to ask questions so unfortunately I am placing it here.

Does this project work with WASM? Specifically Microsoft Blazor.

**Comment by bernardnormier** [comment, marked as answer]
Hi Mark, we currently don't include support for Blazor.

I think it would be reasonably easy to do. It's a matter of adding a new duplex transport for Blazor:
https://docs.icerpc.dev/icerpc/duplex-transport

I assume you'd want to use Blazor to write a web client. What would your web client talk to? An Ice server using ws/wss? An IceRPC server?
